Pocket.xcodeproj/project.pbxproj

```diff
@@ -43,10 +43,8 @@
 		16E4B16F27A3212900E01746 /* Response+factories.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16E4B16E27A3212900E01746 /* Response+factories.swift */; };
 		16EE3F5026CEC80900249AF4 /* PullToRefreshTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16EE3F4F26CEC80900249AF4 /* PullToRefreshTests.swift */; };
 		16FCADBF26CB04AC00906C03 /* ArchiveAnItemTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 16FCADBE26CB04AC00906C03 /* ArchiveAnItemTests.swift */; };
-		3E15B8F0292688EB009A26B6 /* Localizable.strings in Resources */ = {isa = PBXBuildFile; fileRef = 3E15B8F2292688EB009A26B6 /* Localizable.strings */; };
```

Comment:
out of curiosity, why are these removed?

Reply:
These are moved to PocketKit instead of at root level